### DISCUSSION TITLE
Importer does not import monthly credit rate

### DISCUSSION THREAD

**Discussion by NeoZ83-4** [firefly-iii/firefly-iii#7784] [General]
Hello everyone,
first off I want to thank you for this Beautiful Piece of Art that you made with FireFly-iii.
I love it, although I just get started with it.
My First Import of Data went very well.
But I cannot get the Importer to import the monthly credit rate, there are 2 Monthly Payments, that he doesnt like at all.
The Message I get while importing is:
`Line #87:

    (2)transactions.0.destination_id: Source and destination are the same. (original value: "1")
    (2)transactions.0.destination_name: Source and destination are the same. (original value: "(not found)")

Line #88:

    (2)transactions.0.destination_id: Source and destination are the same. (original value: "1")
    (2)transactions.0.destination_name: Source and destination are the same. (original value: "(not found)")`

I did not find the Destination_id & destination_name in the Import Configuration, I also tried the config for the Sparkasse from Github, but no change here.
When needed, I can provide this two lines, but would have to anonymize them.

Kind Regards, and sorry for my hopefully not terrible English
Alex

**Comment by JC5** [comment]
Thanks for the compliments!

Sometimes banks add "special transactions", like interest payments or deposits, that magically 'appear' in your account. If they are saved in the CSV file with the same destination and source account, the data importer will not accept them as transactions because Firefly III cannot handle such transactions.

You will have to create the transaction manually or edit the file so another source / destination account is used.

**Reply by NeoZ83-4** [reply]
Hi, thanks a lot for your quick reply.
I managed to get this two lines imported.
The Error occured because the Bank Account Name matched the "Opposing Account Name", and Firefly 3 didn´t like this (understandably).
